doc/data/cmip_other.rst

```diff
@@ -0,0 +1,160 @@
+
+For either way to download the data, please follow the :ref:`data_download_guideline` to download and move your data to the right place so that it will be sorted to the DKRZ structure.
+   
+If you are a NIRD user, but not a member of this project and would like to request access, contact michaels@met.no.
+
+.. note::
+    NorESM CMIP data !!!  They are stored on nird under /pojects/NS9034K/ (not under /projects/NS9252K). You don't need to download NorESM data to avoid duplication.
+
+.. _data_download_guideline:
+
+Data download guideline
+---------------------
+
+(This guidline is also noted in /cluster/shared/ESGF/README.)
+
+1. Download your data under (recommended, but OK to be elsewhere): ::
+
+    /cluster/shared/ESGF/rawdata/model
+
+2. After all downloaded, invoke the script ``move2autosort.sh`` to move data to ``/cluster/shared/ESGF/rawdata/autosort`` and automatically set correct file permission/ownership:
+
+For example: ::
+
+    $ cd /cluster/shared/ESGF/rawdata
+    $ ./move2autosort.sh "/cluster/shared/ESGF/rawdata/model/mydata*.nc"
+or  ::
+
+    $ ./move2autosort.sh "/cluster/shared/ESGF/rawdata/model/mydatafolder"
+
+.. note::
+    * You can create subfolders to organise the downloaded data
+    * Do NOT download data directly to /cluster/shared/ESGF/rawdata/autosort
+    * The data will start to be automatically sorted to ESGF/DKRZ folder structures within 30 minutes.
+    * The users can NOT invoke the 'raw2dkrz.sh' script anymore (since April 2022).
+
+3. If data are NOT sorted to the desired place, then:
+    - Check if they have been moved to: ``/cluster/shared/ESGF/rawdata/autosort/failed``
+    - If so, check what was going on in the recent log under: ``/cluster/shared/ESGF/rawdata/logs``
+
+.. note::
+    * If you are sure your dataset are correct, you can try to move your data under /cluster/shared/ESGF/rawdata/autosort/failed again back to /cluster/shared/ESGF/rawdata/autosort (simply by 'mv' or 'move2autosort.sh')
+    * Sometimes the HPC will be rebooted, and the crontab job to automatically sort the data will be swipped away. Or some internet problem so that the Synda connections can not be established correctly. 
+    * If there is still problem, please contact yanchun.he@nersc.no to report the error (but not for specific data download requests).
+
+.. _download_with_synda:
+
+Download data with Synda
+------------------------
+
+``Synda`` is installed under Betzy for downloading and managing CMIP data from ESGG.
+
+It is currently installed at `/cluster/shared/ESGF/software/synda`.
+
+(Note, this just serves as an alternative to the tool/scripts `ESGF_download <https://github.com/metno/ESGF_download>`_ by Jans)
+
+Refer to its `documentation <http://prodiguer.github.io/synda/>`_ for introduction and user's guide. But be alerted that its documentation seems not well written and keeps up-to-date.
+
+A simply tutorial on how to use it to download and mange CMIP data.
+
+Activate the tool
+++++++++++++++++
+
```

Comment:
Will this work on Betzy? Or do you want to add something for Betzy? The conda option is not available on Betzy I think, and just typing in this line (ln -s /cluster/shared/ESGF/software/synda/bin/synda ~/local/bin/synda) will give an error since /local/bin/synda does not exist.  I'm not sure how much details people need, but probably more than less :-)

Reply:
Yes, it works for me. I make small fix and update. Could you try it again? thanks

Reply:
preview the changes at: http://ns2345k.web.sigma2.no/people/yanchun/_build/data/cmip_other.html